src/main/java/am/shop/model/Users.java

```diff
@@ -0,0 +1,38 @@
+package am.shop.model;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@ToString
+@EqualsAndHashCode
+public class Users {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @JsonIgnore
+    private Long id;
```

Comment:
In java code model description must be a camel case but for database must be snack case